app/components/TrustedSourceList.js

```diff
@@ -16,9 +16,14 @@ export const TrustedSourceList = ({
   filterAuthoritesByGPSHistory,
   onMenuClick,
 }) => {
-  const isAuthoritySelected = name => {
+  const { t } = useTranslation();
+  const isAuthoritySelected = item => {
```

Comment:
`isAuthoritySelected` seems to be the opposite of the name..

It returns true if the authority is NOT found in the list.

Either change name to `isAuthorityAvailable`, or change the return value to be `return findIndex(...) >= 0`